packages/rke2-snapshot-controller/generated-changes/overlay/crds/groupsnapshot.storage.k8s.io_volumegroupsnapshotclasses.yaml

```diff
@@ -92,3 +92,11 @@ spec:
     served: true
     storage: true
     subresources: {}
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: to not break compatibility when upgrading from previous versions
+          of the CRD
```

Comment:
This line break seems weird to me, won't yaml try to interpret the "of the CRD" as a different attribute?